src/java/src/main/java/com/dnanexus/DXWorkflow.java

```diff
@@ -239,4 +243,101 @@ public Describe getCachedDescribe() {
         return ExecutableRunner.getWorkflowRunnerWithEnvironment(this.getId(), this.env);
     }
 
+    /**
+     * A workflow stage
+     *
+     */
+    public static class DXStage {
```

Comment:
I prefer `DXWorkflow.Stage` to `DXWorkflow.DXStage`... other nested classes also aren't named with a "DX" prefix.